src/lib389/lib389/cli_conf/replication.py

```diff
@@ -335,13 +333,10 @@ def get_repl_winsync_status(inst, basedn, log, args):
     replicas = Replicas(inst)
     replica = replicas.get(args.suffix)
     if args.bind_passwd_file is not None:
-        passwd = get_passwd_from_file(args.bind_passwd_file)
-    elif args.bind_passwd_prompt:
-        passwd = _get_arg(None, msg=f"Enter password for ({args.bind_dn})", hidden=True, confirm=True)
-    else:
-        passwd = args.bind_passwd
-
-    status = replica.status(binddn=args.bind_dn, bindpw=passwd, winsync=True)
+        args.bind_passwd = get_passwd_from_file(args.bind_passwd_file)
+    if args.bind_passwd_prompt or args.bind_dn is None or args.bind_passwd is None:
+        pw_and_dn_prompt = True
+    status = replica.status(binddn=args.bind_dn, bindpw=args.bind_passwd, winsync=True, pwprompt=pw_and_dn_prompt)
```

Comment:
The function 'status' does not prompt password for windows RA. IIUC either we are not modifying get_repl_winsync_status or in 'status' we need to prompt the password for any remote.

Reply:
I've changed it here because we already had the bind-password options.
Yes, now, it does nothing, as we don't get the lag time for winsync agreements:

```
            lag_time = "Not available for Winsync agreements"
            status = "Not available for Winsync agreements"
```
       
I'm not sure if the feature will be added in the future though. If it will, then my change will come in handy.
If not, we can deprecate the password arguments altogether from the get_repl_winsync_status.

But it's not the goal of this PR.
       
That's why I added it to winsync, too.